ni_measurementlink_service/_dotenvpath.py

```diff
@@ -44,7 +45,7 @@ def _get_caller_path() -> Optional[Path]:
     for frame, _ in traceback.walk_stack(inspect.currentframe()):
         if frame.f_code.co_filename:
             module_path = Path(frame.f_code.co_filename)
-            if module_path.exists() and not _is_relative_to(module_path, nims_path):
+            if os.path.exists(module_path) and not _is_relative_to(module_path, nims_path):
```

Comment:
It's annoying when subtle differences like this happen between Python versions.

@bkeryan, should we keep this like this? (I believe the Path object will be converted back to a string - non-optimal but not a huge hit to performance unless called many times) 
or make a helper method like we did for `_is_relative_to` that provides a try/except (although, that require a program stack operations and will also cost us a few clock cycles), 
or directly reference the str from above (probably by storing off `frame.f_code.co_filename`)?

Reply:
I don't like mixing pathlib and os.path, so I would prefer doing one of the following:
- Rewrite this entire line to use os.path. Is there an os.path one-liner equivalent to is_relative_to?
- Add a helper method and use try/except to ignore OSError when e.errno == errno.EINVAL.

Reply:
> Is there an os.path one-liner equivalent to is_relative_to?

I did a little web search but couldn't find any equivalent methods.

> Add a helper method and use try/except to ignore OSError when e.errno == errno.EINVAL.

I used the `sys.version` approach as described in the other thread since I felt it was comparatively more expressive.

Reply:
Ok, the updated code is fine with me. I'm not concerned about the performance aspect because this should only be used for one-time initialization code and it's not used for Python 3.10+.